backend/controller/ingress/ingress.go

```diff
@@ -60,40 +60,33 @@ func matchSegments(pattern, urlPath string, onMatch func(segment, value string))
 }
 
 func ValidateCallBody(body []byte, verbRef *schema.VerbRef, sch *schema.Schema) error {
-	var requestMap map[string]any
-	err := json.Unmarshal(body, &requestMap)
-	if err != nil {
-		return fmt.Errorf("HTTP request body is not valid JSON: %w", err)
-	}
-
 	verb := sch.ResolveVerbRef(verbRef)
 	if verb == nil {
 		return fmt.Errorf("unknown verb %s", verbRef)
 	}
 
-	dataRef := verb.Request
+	var requestMap map[string]any
+	err := json.Unmarshal(body, &requestMap)
+	if err != nil {
+		return fmt.Errorf("HTTP request body is not valid JSON: %w", err)
+	}
 
-	return validateRequestMap(dataRef, []string{dataRef.String()}, requestMap, sch)
+	return validateRequestMap(verb.Request, []string{verb.String()}, requestMap, sch)
```

Comment:
Do you want `verb.String()` here? Won't that be like `add(AddRequest) AddResponse` type thing?

Reply:
LOL 🤦 yeah :)